### PR TITLE
[OCPBUGS-45747]: Fixing command in HCP docs

### PR DESCRIPTION
**Pull request by lahinson**
Version(s): 4.17+

Issue: https://issues.redhat.com/browse/OCPBUGS-45747

Link to docs preview: https://86235--ocpdocs-pr.netlify.app/openshift-enterprise/latest/hosted_control_planes/hcp-manage/hcp-manage-bm.html#hcp-bm-add-np_hcp-manage-bm

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/hcp-bm-add-np.adoc [hunk 1]
```diff
@@ -19,7 +19,7 @@ $ hcp create nodepool agent \
   --cluster-name <hosted_cluster_name> \// <1>
   --name <nodepool_name> \// <2>
   --node-count <worker_node_count> \// <3>
-  --agentLabelSelector '{"matchLabels": {"size": "medium"}}' <4>
+  --agentLabelSelector size=medium <4>
```
Comment (shipsing) [inline modules/hcp-bm-add-np.adoc:22]: Should the description be 
<4> The `--agentLabelSelector` flag is optional. The node pool uses agents with the `size=medium` label ?

Do we need to update the callout4 description accordingly?

Comment (lahinson) [reply modules/hcp-bm-add-np.adoc:22]: Yes! Good catch. Thanks!

**Pull request merged**